2.0/UnlitTest/README.md

```diff
@@ -0,0 +1,21 @@
+# Unlit Test
+
+## Screenshot
+
+![screenshot](screenshot/screenshot_large.jpg)
+
+## Description
+
+This model tests the [`KHR_materials_unlit`](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_unlit) extension.  There are two objects, one orange-ish (`#FF7F00` in sRGB space) and one blue-ish (`#007FFF` in sRGB space).  The visible surface of these two objects should appear as a uniform color, so much so that the 3D effect is intentionally lost.
+
+## Problem: Lighting applied to un-lit model
+
+![screenshot](screenshot/unlit_test_fail.jpg)
```

Comment:
minor: screenshots should say `KHR_materials_unlit` (plural)

Reply:
Thanks!